keystore/src/entities/platform/generic/proteus/identity.rs

```diff
@@ -143,7 +143,7 @@ impl EntityBase for ProteusIdentity {
 
     async fn delete(
         conn: &mut Self::ConnectionType,
-        _ids: &[crate::entities::StringEntityId],
+        _ids: crate::entities::StringEntityId<'_>,
```

Comment:
this should be `id` if it's just one

Reply:
fixed by https://github.com/wireapp/core-crypto/pull/695/commits/7beae609399c01513b60bc51267a1be369c2048e